aries-site/src/examples/templates/global-header/components/SidebarLayer.js

```diff
@@ -0,0 +1,123 @@
+import React, { useContext, useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Avatar,
+  Box,
+  Button,
+  Header,
+  Footer,
+  Layer,
+  Nav,
+  ResponsiveContext,
+  Text,
+} from 'grommet';
+import { Aruba, FormClose, Menu } from 'grommet-icons';
+
+export const SidebarLayer = () => {
```

Comment:
Another naming question for future implementers. What does this layer really represent?

AccountMenu?